### DISCUSSION TITLE
Turborepo now supports nodeLinker: pnp in yarn berry?

### DISCUSSION THREAD

**Discussion by Brew-Brew** [vercel/turborepo#7055] [Help]
i heard that turborepo doesn't support pnp mode, but it works on my repo. can i hear any updates of this..?

**Comment by mehulkar** [comment]
Where did you hear that and what update are you looking for? I'm not aware of any claims we're making either way!

**Reply by kyun** [reply]
https://github.com/vercel/turbo/discussions/334 
I am confused.

**Reply by mehulkar** [reply]
Thank you for the reference, I'll chase down a more definitive answer and get our [support policy](https://turbo.build/repo/docs/support) updated
